_data/sites.json

```diff
@@ -19181,6 +19181,16 @@
         ]
     },
 
+    {
+        "name": "UXPin",
+        "url": "https://www.uxpin.com/docs/account/closing-your-account/",
+        "difficulty": "hard",
+        "notes": "You must call them to close your account.",
+        "domains" : [
+            "uxpin.com"
```

Comment:
```suggestion
            "uxpin.com",
            "app.uxpin.com",
            "files.uxpin.com",
            "get.uxpin.com",
            "showcase.uxpin.com"
```

Reply:
Sorry, I wasn't aware. How'd you discover those?

Reply:
> Sorry, I wasn't aware. How'd you discover those?
@SteffenGivard 

I just scrolled to the very bottom of the homepage www.uxpin.com and briefly looked at everything they've linked there. (in the black footer section)